docker-compose.yml

```diff
@@ -42,9 +40,7 @@ services:
       - mongo
 
   reverse-proxy:
-    # TODO(dmu) CRITICAL: Replace image identifiers to those hosted on a docker repository
-    #                     https://thenewboston.atlassian.net/browse/BC-70
-    image: node-reverse-proxy:current
+    image: ghcr.io/thenewboston-developers/reverse-proxy:latest
```

Comment:
should not it be `image: ghcr.io/thenewboston-developers/node-reverse-proxy:latest` ?